integration/jackson/src/main/java/com/blazebit/persistence/integration/jackson/EntityViewAwareObjectMapper.java

```diff
@@ -136,12 +136,7 @@ public ObjectReader readerFor(JavaType javaType) {
         if (Collection.class.isAssignableFrom(javaType.getRawClass())) {
             return objectMapper.readerFor(javaType);
         } else {
-            return readerFor(javaType.getRawClass());
+            return objectMapper.readerFor(javaType.getRawClass());
         }
     }
-
-    public ObjectReader readerFor(Class<?> type) {
```

Comment:
Please revert this. The idea of the method is that a user should be able to provide a custom object reader.